tests/collections/db.test.ts

```diff
@@ -84,14 +89,20 @@ for (const testClient in testClients) {
       });
     });
 
-    describe('dropDatabase', () => {
+    describe('dropDatabase', function(this: Mocha.Suite) {
+      const suite = this;
       after(async () => {
-        const keyspaceName = parseUri(process.env.JSON_API_URI).keyspaceName;
+        if(isAstra){
+          return;     
+        }
+        const keyspaceName = parseUri(dbUri).keyspaceName;
         await createNamespace(astraClient.httpClient, keyspaceName);
       });
-
       it('should drop the underlying database (AKA namespace)', async () => {
-        const keyspaceName = parseUri(process.env.JSON_API_URI).keyspaceName;
+        if(isAstra){
+          suite.ctx.skip();
```

Comment:
Will `dropDatabase()` be supported in Astra?

Reply:
We are getting a PERMISSION DENIED error even with a Database Administrator access, so skipping these tests for Astra.